rqt_gui/src/rqt_gui/ros_plugin_provider.py

```diff
@@ -82,12 +82,14 @@ def load(self, plugin_id, plugin_context):
             qCritical('RosPluginProvider.load(%s): raised an exception:\n%s' % (plugin_id, e))
             return None
         except Exception as e:
-            qCritical('RosPluginProvider.load(%s) exception raised in __builtin__.__import__(%s, [%s]):\n%s' % (plugin_id, attributes['module_name'], attributes['class_from_class_type'], traceback.format_exc()))
+            qCritical('RosPluginProvider.load(%s) exception raised in __builtin__.__import__(%s, [%s]):\n%s' % (
```

Comment:
break into more lines

Reply:
I would prefer to keep changes limited to autopep8